### PR TITLE
Saas 817 list of support tickets UI tests

### PR DESCRIPTION
**Pull request by peterSirotnak**
Pull request for [SAAS-817](https://jira.percona.com/browse/SAAS-817)

### CODE REVIEW

File: cypress/integration/customer/dashboard/dashboard.spec.js [hunk 1]
```diff
@@ -26,4 +27,50 @@ context('Dashboard Tests for customers', () => {
...
+      cy.log(`Running test for ${user.email} user`);
+      cy.intercept('POST', '**/tickets:search').as('userTickets');
+      cy.loginByOktaApi(user.email, user.password);
+      cy.findAllByTestId(commonPage.locators.sideMenuLink).get('a[href*="/dashboard"]').click();
+      //Table header exists with correct fields
+      cy.findByTestId(dashboardPage.locators.tableHeader)
+        .get('th')
+        .then((cells) => {
+          expect(cells.length).to.be.equal(7);
+          for (let i = 0; i < cells.length; i++) {
```
Comment (yurkovychv) [inline cypress/integration/customer/dashboard/dashboard.spec.js:42]: it's better to use [expect](https://docs.cypress.io/guides/references/assertions#BDD-Assertions) here too. Example: `expect([1, 2, 3]).to.include.members([3, 2])`.

File: cypress/integration/customer/dashboard/helper.js [hunk 2]
```diff
@@ -8,3 +10,8 @@ export const createOrgAndAddUsers = (users) => {
     }),
   );
 };
+
+export const isTicketTableHeaderTextCorrect = (cellText, position) => {
```
Comment (yurkovychv) [inline cypress/integration/customer/dashboard/helper.js:14]: I believe this one is not needed if you will use `expect` from comment above

**Review by yurkovychv — CHANGES_REQUESTED** [review]

**Comment by yurkovychv** [comment]
@peterSirotnak Also please add `non-SN users that are members of an organization linked with SN` test

File: cypress/integration/customer/dashboard/dashboard.spec.js (hunk 1 above)
Comment (peterSirotnak) [reply cypress/integration/customer/dashboard/dashboard.spec.js]: Okay, fixed.

File: cypress/integration/customer/dashboard/helper.js (hunk 2 above)
Comment (peterSirotnak) [reply cypress/integration/customer/dashboard/helper.js]: Yes that is correct, removed.

**Comment by peterSirotnak** [comment]
> @peterSirotnak Also please add `non-SN users that are members of an organization linked with SN` test. And test case IDs to test summary

Added that test and also added waiting for dashboard page to fully load. Ticket table is rendered after loading spinners are not present.

File: cypress/integration/customer/dashboard/dashboard.spec.js [hunk 3]
```diff
@@ -26,4 +29,65 @@ context('Dashboard Tests for customers', () => {
       cy.removeCurrentUserAccessToken();
     });
   });
+
+  it('Verify Percona Customer user is able to see tickets created for his org', () => {
```
[the remark this answers is not in the document]
Comment (peterSirotnak) [reply cypress/integration/customer/dashboard/dashboard.spec.js]: Fixed

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**